### PR TITLE
added aws-sso to devcontainer

### PR DESCRIPTION
**Pull request by jamesstottmoj**
Allows use of aws cli in dev container

### CODE REVIEW

**Review by michaeljcollinsuk — APPROVED** [review]
LGTM

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
       "version": "3.11"
     },
     "ghcr.io/devcontainers/features/docker-in-docker:2": {},
-    "./features/src/postgresql": {}
+    "./features/src/postgresql": {},
+    "ghcr.io/ministryofjustice/devcontainer-feature/aws:0": {}
```
Comment (michaeljcollinsuk) [inline .devcontainer/devcontainer.json:13]:
```suggestion
    "ghcr.io/ministryofjustice/devcontainer-feature/aws:0": {},
    "ghcr.io/ministryofjustice/devcontainer-feature/static-analysis:0": {},
```
Adds static analysis, as per @Gary-H9's suggestion

**Review by michaeljcollinsuk — CHANGES_REQUESTED** [review]

**Review by michaeljcollinsuk — DISMISSED** [review]

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -9,7 +9,9 @@
       "version": "3.11"
     },
     "ghcr.io/devcontainers/features/docker-in-docker:2": {},
-    "./features/src/postgresql": {}
+    "./features/src/postgresql": {},
+    "ghcr.io/ministryofjustice/devcontainer-feature/aws:0": {},
+    "ghcr.io/ministryofjustice/devcontainer-feature/static-analysis:0": {},
```
Comment (michaeljcollinsuk) [inline .devcontainer/devcontainer.json:14]:
```suggestion
    "ghcr.io/ministryofjustice/devcontainer-feature/static-analysis:0": {}
```
Super linter fix

**Review by Gary-H9 — APPROVED** [review]
LGTM

**Pull request merged**